.github/workflows/ghcr-build.yml

```diff
@@ -355,7 +355,7 @@ jobs:
       - name: Install poetry via pipx
         run: pipx install poetry
       - name: Install Python dependencies using Poetry
-        run: make install-python-dependencies
+        run: make install-python-dependencies INSTALL_PLAYWRIGHT=0
```

Comment:
doesn't the env var need to come before the command?

Reply:
For most commands yes. This is a make feature:

https://www.gnu.org/software/make/manual/html_node/Overriding.html